src/lib/list/list.spec.ts

```diff
@@ -144,6 +146,30 @@ describe('MatList', () => {
     items.forEach(item => expect(item._isRippleDisabled()).toBe(true));
   });
 
+  it('should create an action list', () => {
+    let fixture = TestBed.createComponent(ActionListWithoutType);
```

Comment:
Prefer `const` over `let` whenever the variable isn't going to be assigned to something else